src/api/ledger/utils.js

```diff
@@ -105,6 +105,28 @@ function hasCompleteLedgerRange(remote: Remote, minLedgerVersion?: number,
     maxLedgerVersion || remote.getLedgerSequence());
 }
 
+function hasLedger(remote: Remote, ledger: ?number) {
+  return remote.getServer().hasLedger(ledger);
+}
+
+function isNonContiguousLedgerRange(remote: Remote, minLedgerVersion: ?number,
+                                    maxLedgerVersion: ?number
+): boolean {
+  if (!minLedgerVersion || !maxLedgerVersion) {
+    return false;
```

Comment:
But if you use my suggestion above you could delete this whole function anyways.